modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts

```diff
@@ -83,6 +83,86 @@ export interface SiteConfig<Config> {
     config: Config;
 }
 
+export type DefaultHeaders = {
```

Comment:
There are a few problems with these headers definitions 
- the list is not complete (and cannot be)
- it enforces headers characters case
- it is called default,  but it is not 

I suggest to remove it